docs/src/API_GRPC_Explanation.md

```
@@ -151,18 +316,60 @@ The `GrpcAuthentication` object supports two modes:
+     ```toml
+     grpc_authentication = { username = "admin", password = "mysecurepassword" }
+     ```
+
+### 4. **Save the File**
+   - After making the changes, save the `config.toml` file.
+
+### 5. **Restart the Wallet**
+   - Restart your Tari wallet application to apply the updated configuration.
+
+### 6. **Client-Side Configuration**
+   - Ensure your gRPC client connects using basic authentication. For example, in JavaScript:
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 0; Actual: 3

(MD007, ul-indent)